src/main/java/com/instinctools/clientaccountmanager/services/AccountService.java

```diff
@@ -0,0 +1,11 @@
+package com.instinctools.clientaccountmanager.services;
+
+import com.instinctools.clientaccountmanager.model.Account;
+
+public interface AccountService {
```

Comment:
Why we need separate account service for this task? I think we can stay only with ClientService with methods like:
`updateAccount`
`createAccount`
`deleteAccount`